src/rajAJun20/StringNumbersSum.java

```diff
@@ -0,0 +1,22 @@
+package rajAJun20;
+
+public class StringNumbersSum {
+
+	int showSumOfNumbersInString(String str) {
+		int sum = 0 ;
+		
+		for(int index = 0 ; index < str.length() ; index++) {
+			if(Character.isDigit(str.charAt(index))) {
+				//sum = sum + Character.getNumericValue(str.charAt(index));
+				sum = sum + Integer.parseInt(String.valueOf(str.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));